index.js

```diff
@@ -0,0 +1,127 @@
+var Role;
```

Comment:
js files should not be part of github. Add the whole transpiled code folder into gitignore